### PR TITLE
Save the Swing (Add swing to Glee Club and Shrimp Shuffle and add Japanese versions of those games), also mole warnings and Rhythmove Dungeon.

### PR DESCRIPTION
**Pull request by GuardedLolz**
### Prerequisites
* [X] This pull request follows the code style of the project
* [X] I have tested this feature thoroughly WITH THE USE of the load-all-sounds starter
  * https://www.youtube.com/watch?v=A9WjMdo2b4s
* [X] I have read and understood the [contribution guidelines](CONTRIBUTING.md)

### Changes Proposed in this Pull Request
See above.

Thanks to ChorusSquid for databasing Rhythmove Dungeon.
...

### CODE REVIEW

File: core/assets/sounds/cues/gleeClubEn/data.json [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+      "name": "scream"
+    },
+    {
+      "id": "gleeClubEn/baton",
+      "deprecatedIDs": [
+        "gleeClub/baton"
+      ],
+      "duration": 1,
+      "name": "baton - lift",
+      "fileExtension": "wav"
```
Comment (chrislo27) [inline core/assets/sounds/cues/gleeClubEn/data.json]: Convert to ogg vorbis please

**Review by chrislo27 — CHANGES_REQUESTED** [review]
Misleading title? I don't see any swing in Glee Club nor do I think it's necessary if present.

**Comment by GuardedLolz** [comment]
toGE...therNOW!

**Comment by GuardedLolz** [comment]
Dude. The video. It exists for a reason.

**Comment by chrislo27** [comment]
Please avoid being rude. I don't stalk this pull request day and night, and every time you make a PR initially you never provide a video.

**Pull request merged**